src/index.html

```diff
@@ -5,8 +5,288 @@
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div class="card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="iMac 27 Retina"
+          class="card__img"
+          >
+        <p class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__rate-reviews">Reviews: 5</div>
+        </div>
+        <div class="card__price-container">
+          <p class="card__price-label">Price:</p>
+          <p class="card__price">$2,199</p>
+        </div>
+        <a
+          href="#"
+          data-qa="hover"
+          class="card__button"
+          >Buy </a>
+      </div>
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="iMac 27 Retina"
+          class="card__img"
+          >
+        <p class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__rate-reviews">Reviews: 5</div>
+        </div>
+        <div class="card__price-container">
+          <p class="card__price-label">Price:</p>
+
+          <p class="card__price">$2,199</p>
+        </div>
+        <a
+          href="#"
+          data-qa="hover"
+          class="card__button"
+          >Buy </a>
+      </div>
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="iMac 27 Retina"
+          class="card__img"
+          >
+        <p class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__rate-reviews">Reviews: 5</div>
+        </div>
+        <div class="card__price-container">
+          <p class="card__price-label">Price:</p>
+
+          <p class="card__price">$2,199</p>
+        </div>
+        <a
+          href="#"
+          data-qa="hover"
```

Comment:
this data-qa should be only for 1st card, please remove it everywere else

Reply:
Done.